src/RayTracer/Public/js/drag_drop.js

```diff
@@ -59,26 +71,31 @@ function showObjectDrag(){
 
 function bindListeners(){
   $(".svg-object").mousedown(function(e){
-    module.exports.dragged_id = e.target.id;
+    dragged_id = e.target.id;
     //Have to disable pointer events and all objects except for the one I am dragging
     //this is because if I drag the target object over an object that is rendered above it, the focus of the event changes!
     $(".svg-object").not(e.target).css( 'pointer-events', 'none' );
     $("#svg").mousemove(function(e){
-      if(e.target.id != module.exports.dragged_id){
+      if(e.target.id != dragged_id){
         $(".svg-object").trigger("mouseup");
       }
 
+      const dimension = ($('input[name=chosen-view]:checked')[0].value) == "Side" ? "y" : "z"
+
+
       //Calculate position of the cursor in reference to the SVG
       let x_new = e.pageX - $("#svg").position().left;
-      let y_new = e.pageY - $("#svg").position().top;
+      let yz_new = e.pageY - $("#svg").position().top;
```

Comment:
yz is confusing. can it be called something else so its more clear?

Reply:
Thanks, that's a good point. I've added a comment to make it clearer